Hudl.Mjolnir.Tests/Command/Attribute/NewCommandAttributeAndProxyTestsIgnoringTimeouts.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using Hudl.Mjolnir.Command.Attribute;
+using Hudl.Mjolnir.Tests.Helper;
+using Xunit;
+
+namespace Hudl.Mjolnir.Tests.Command.Attribute
+{
+    public sealed class NewCommandAttributeAndProxyTestsIgnoringTimeouts : TestFixtureIgnoreTimeouts
```

Comment:
We can do that, just means running the TestFixtureIgnoreTimeouts and TestFixture constructor logic at the start of each test. 
A lot of changes but In the spirit of the xunit philosophy maybe we should do that.